.github/workflows/tests_linux_x86.yml

```diff
@@ -79,6 +79,7 @@ jobs:
       - name: Build and run unit tests
         run: |
             cmake . -BBuild \
+              -DPLLGPU_ENABLE_MPI=OFF \
```

Comment:
This is the default, right? Do we need it here?

Reply:
Removed! Thanks